libs/coin-framework/src/env.ts

```diff
@@ -0,0 +1,790 @@
+import mapValues from "lodash/mapValues";
```

Comment:
ok sorry for missing this before. But I think this file don't belong to `coin-framework` as it contains envs that are not even related to coins.
 I suggest that we make a dedicated lib `env` so we can use `@ledgerhq/env` instead. (quid to guarantee a dedup of this lib, because it's stateful, we have a memory state that changes with setEnv)

Reply:
Unfortunately, this file contains URL for coin family request. There are also variable like `DEBUG_HTTP_RESPONSE` used inside the `network.ts` file.
I fully agree that we should put this file elsewhere. But I afraid the consequences of refactoring it right now will bring too much work for this PR.
WDYT? @gre

Reply:
indeed. can you prepend in that file a comment?

```
// TODO: this file will be moved to a @ledgerhq/env library
```

I can help working on this after the PR lands.